examples/meta/src/multiclass/multiclass_machine.sg.in

```diff
@@ -0,0 +1,34 @@
+CSVFile f_feats_train("@SHOGUN_DATA@/classifier_4class_2d_linear_features_train.dat")
+CSVFile f_feats_test("@SHOGUN_DATA@/classifier_4class_2d_linear_features_test.dat")
+CSVFile f_labels_train("@SHOGUN_DATA@/classifier_4class_2d_linear_labels_train.dat")
+
+#![create_features]
+Features features_train = features(f_feats_train)
+Features features_test = features(f_feats_test)
+Labels labels_train = labels(f_labels_train)
+#![create_features]
+
+#![create_kernel]
+Kernel kernel_gaussian = kernel("GaussianKernel", log_width=2.1)
```

Comment:
"kernel"
Also choose a better log_width, like 0

Reply:
wdym "kernel"?

Reply:
Kernel kernel = kernel("GaussianKernel", log_width=0)

Reply:
I think?

Reply:
I will make it `Kernel kernel = kernel("GaussianKernel", log_width=0.0)` and change it later if needed